src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/ResponsibilitiesLogicTests.kt

```diff
@@ -72,12 +77,77 @@ class ResponsibilitiesLogicTests : MontaguTests()
             on { touchstoneVersions } doReturn mockTouchstoneVersions
         }
 
-        val sut = RepositoriesResponsibilitiesLogic(mock(), mock(), touchstoneRepo)
+        val sut = RepositoriesResponsibilitiesLogic(mock(), mock(), touchstoneRepo, mock(), mock(), mock())
 
         Assertions.assertThatThrownBy {
             sut.validateResponsibilityPath(path, statusList)
         }.isInstanceOf(UnknownObjectError::class.java).hasMessageContaining("Unknown touchstone-version with id 'touchstone-1'")
 
     }
 
-}
\ No newline at end of file
+    @Test
+    fun `can get data`()
+    {
+        val now = Instant.now()
+        val then = now.minusMillis(1)
+        val responsibilitiesRepo = mock<ResponsibilitiesRepository> {
+            on { getResponsibilitiesWithCommentsForTouchstone("202002rfp-1") } doReturn listOf(
+                    ResponsibilitySetWithComments("202002rfp-1", "VIMC", ResponsibilityComment("Note for VIMC for 202002rfp-1", "test.user", now), listOf(
+                            ResponsibilityWithComment("scenario-1", ResponsibilityComment("Note for VIMC, Campaign, Cholera for 202002rfp-1", "test.user2", then))
+                    ))
+            )
+            on { getResponsibilitiesForTouchstone("202002rfp-1") } doReturn listOf(
+                    ResponsibilitySet("202002rfp-1", "VIMC", ResponsibilitySetStatus.INCOMPLETE, listOf(
+                            Responsibility(
+                                    Scenario("scenario-1", "Default - Campaign", "Cholera", listOf("202002rfp")),
+                                    ResponsibilityStatus.EMPTY,
+                                    emptyList(),
+                                    BurdenEstimateSet(
+                                            1,
+                                            now,
+                                            "test.user",
+                                            BurdenEstimateSetType(BurdenEstimateSetTypeCode.CENTRAL_SINGLE_RUN, "time varying CFR"),
+                                            BurdenEstimateSetStatus.PARTIAL,
+                                            emptyList(),
+                                            null
+                                    )
+                            )
+                    ))
+            )
+        }
+        val expectationMapping = ExpectationMapping(CountryOutcomeExpectations(77, "", 2001..2002, 97..99, CohortRestriction(), listOf(Country("UK", "United Kingdom")), listOf()), listOf(""), "Cholera")
+        val expectationsRepo = mock<ExpectationsRepository> {
+            on { getExpectationsForResponsibility(42) } doReturn expectationMapping
+        }
+        val burdenEstimateRepo = mock<BurdenEstimateRepository> {
```

Comment:
Again, maybe one or two comments in this test about where the expectations for the less obvious parts of the results are coming from e.g. the count of missing estimates.